src/posts/pitfalls-of-accessible-components.md

```diff
@@ -0,0 +1,90 @@
+
+### Focus Order
+
+Make sure that every page you tab through navigates logically. This is easier if you don't use a tab index with a value higher than 0. Creative application of CSS grid or flexbox can also cause the visual order to differ from that in the code.
+Do you want to make it completely easy for your visitor? Add a skip link. That is a (often hidden) link at the beginning of the page that allows you to "jump" to the content.
+
+## Difficult to test
+
+### Variations of components
+
+The more features a component has, the greater the chance that it will go wrong. If a component has a boolean attribute, dark mode and two responsive zoom levels, then you quickly end up with (2x2x3=8) eight variations that all need to be tested and maintained. So pay attention to this! If everyone around you has default dark mode on, you can count on issues in light mode.
+
+### Composition
+
+As soon as you start combining components, you can also foresee problems. You would prefer, for example, that forms have been developed and bundled as a complete system. You often combine the same components in forms. If they are delivered separately, unexpected (and untested) situations may arise. Another point of attention!
+Related to this is also the topic of relationships. Everything you tie together yourself is prone to errors. If you have a table of contents at the top of your page, you should pay close attention to how it is connected to the content of your page. Or how about a form (field) and an error message or instruction? You want relationships to be clear both visually and in code.
+
+<figure role="figure" aria-label="The right parts in the wrong order.">
+	<img alt="A projector is attached to a wall. A projection screen is rolled up on the wall above it. The screen can't be rolled out, making it so the projector can't project on it." src="/img/posts/pitfalls-of-accessible-components/projector-and-screen.jpg" />
+	<figcaption>The right parts in the wrong order.</figcaption>
+</figure>
+
+
+### Website-wide
+
+As mentioned earlier, components are good for consistency. If your set of components is not yet very mature or complete, this can still be a point of improvement.
+For example, focus states can still differ between pages and components if this is not well thought out and implemented centrally.
+Another point for improvement in the broader structure is often the navigability. You want pages to be found and accessed in more than one way. Think of a search function or sitemap next to your menu structure. Typically something you can't solve with a component. Make sure all your pages are part of this and are clearly represented.
```

Comment:
```suggestion
As mentioned earlier, components are good for consistency. If your set of components is not yet very mature or complete, this can still be a point of improvement.

For example, focus states can still differ between pages and components if this is not well thought out and implemented centrally.

Another point for improvement in the broader structure is often the navigability. You want pages to be found and accessed in more than one way. Think of a search function or sitemap next to your menu structure. Typically something you can't solve with a component. Make sure all your pages are part of this and are clearly represented.
```

Newlines for new paragraphs.